programs/svm-spoke/src/state/instruction_params.rs

```diff
@@ -0,0 +1,12 @@
+use anchor_lang::prelude::*;
+
+use crate::RelayerRefundLeaf;
+
+#[account]
+#[derive(InitSpace)]
+pub struct ExecuteRelayerRefundLeafParams {
+    pub root_bundle_id: u32,
+    pub relayer_refund_leaf: RelayerRefundLeaf,
+    #[max_len(0)]
```

Comment:
Why do we need this constrain?

Reply:
its a declaration to help calculate initial space when doing account initialization. I'll add required comments